app/src/main/res/values-de/strings.xml

```diff
@@ -45,7 +45,7 @@
     <string name="menu_assistants">Assistenten</string>
     <string name="menu_calendar">Kalender</string>
     <string name="menu_maps">Karten</string>
-    <string name="menu_navigation">Steuerung</string>
```

Comment:
Translations must be managed separately in CrowdIn, it gets confused when the file changes unexpectedly.

Reply:
Reverted

Reply:
https://crowdin.com/editor/androidautoidrive/47/en-de?view=comfortable&filter=basic&value=0# I copied your translations in, feel free to update any others you can think of! It looks like the translation is pretty complete tho, so I'll probably merge the [translation PR](https://github.com/BimmerGestalt/AAIdrive/pull/861/files) this weekend.

Reply:
I did one small change on Crowdin. It should be good to merge. In future I'll do a complete rework of the translations to get the same "speech style" for all strings. But for now everything is done.